dart/test/sentry_client_test.dart

```diff
@@ -613,6 +641,38 @@ SentryEvent? eventProcessorDropEvent(SentryEvent event, {dynamic hint}) {
 }
 
 class Fixture {
+  Fixture() {
+    options = SentryOptions(dsn: fakeDsn);
+    options.transport = MockTransport();
+
+    scope = Scope(options)
+      ..user = user
+      ..transaction = transaction
+      ..fingerprint = fingerprint
+      ..addBreadcrumb(crumb);
+
+    event = SentryEvent(
+      level: SentryLevel.warning,
+      transaction: eventTransaction,
+      user: eventUser,
+      fingerprint: eventFingerprint,
+      breadcrumbs: eventCrumbs,
+    );
+  }
+
+  late SentryOptions options;
+  late Scope scope;
```

Comment:
this sounds more like a createScope(): Scope method than a global field in the fixture

Reply:
https://github.com/getsentry/sentry-dart/pull/467/files#r638548589